### ISSUE TITLE
Differences compared to TESS_PRF

### ISSUE THREAD

**Issue by keatonb** [lightkurve/lkprf#6]
I have my own package for interpolating PRFs for TESS: https://github.com/keatonb/TESS_PRF

lkprf output has some slight differences worth recording here if people switch from TESS_PRF to lkprf.

```python
#Compare interpolated PRF models from lkprf and TESS_PRF
import numpy as np
import matplotlib.pyplot as plt

import lkprf
import PRF #TESS_PRF

# random source position example
colnum = 113 # middle of TPF
rownum = 298 # middle of TPF
sector = 13 # TESS_PRF will use later PRF models
camera = 1
ccd = 1

# interpolate with lkprf
prf1 = lkprf.TESSPRF(camera=camera, ccd=ccd).evaluate(targets=[(rownum, colnum)], origin=(rownum-5, colnum-5), shape=(11, 11))[0]

# interpolate with TESS_PRF
prf2 = PRF.TESS_PRF(camera,ccd,sector,colnum,rownum).locate(5.0, 5.0, (11,11))

#plot results
fig,axs = plt.subplots(1,3,figsize=(10,3))
axs[0].imshow(np.log10(prf1), origin='lower') # lkprf (log10)
axs[0].set_title("lkprf (log)")
axs[1].imshow(np.log10(prf2), origin='lower') # TESS_PRF (log10)
axs[1].set_title("TESS_PRF (log)")

diff = (prf1-prf2) #difference
im = axs[2].imshow(diff, origin='lower')
axs[2].set_title("difference")
plt.colorbar(im, location='right')

plt.tight_layout()
plt.show()
```
<img width="957" alt="image" src="https://github.com/user-attachments/assets/f9a6d19a-0376-4a7c-ba24-b62b82836425">

Differences in output in this example are ~7% in the brightest pixel. A bit of a dipole pattern to the difference image suggesting a centroid offset between the two codes. 

I have no confidence that TESS_PRF is correct, but likely worth trying to find the source of the difference.

There are some minor differences in execution speed that are probably negligible for most use cases. I love that lkprf can sample many source positions in one move!!

**Comment by Nschanche** [comment]
Hi @keatonb , thanks for pointing out this issue! lkprf was using a weighted average of all supersampled PRFs for a given camera/ccd. I've put in PR #8 that uses a weighted average of only the 4 nearest measurements. This better reproduces the shape of the PRF. However, there is still a discrepancy, possibly related to the pixel offset I can look into further. 

<img width="961" alt="prf_comparison" src="https://github.com/user-attachments/assets/214cb51f-4c18-4c3f-ae2c-7eb86da03f9a">

**Comment by Nschanche** [comment]
I've merged in PR #8 and ran the above test using the updated lkprf. There is still a slight resiudual difference between the two models, but with a much smaller magnitude. I think these are due to the different interpolation methods. 

![keaton_comp](https://github.com/user-attachments/assets/433ee0f4-3f09-4510-946e-7fc16250b2ac)

**Comment by Nschanche** [comment]
Closing this issue, but please do let me know if you notice any other issues or discrepancies!
